opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -140,12 +140,11 @@ public SELF user_logged_in_into_opba_as_opba_user_with_credentials_using_fintech
                 .getQueryParams()
                 .getFirst(REDIRECT_CODE_QUERY);
 
-        ExtractableResponse<Response> response = RestAssured
+        ExtractableResponse<Response> response =  withSignatureHeaders(RestAssured
```

Comment:
This function is called within the user session (from UI), not Fintech - it is protected only by users' cookie and not by request signing - no signature headers here

Reply:
Removed signature header adding for non-fintech requests